esmvaltool/recipes/recipe_capacity_factor_wp7.yml

```diff
@@ -0,0 +1,60 @@
+
+#
+# References:
+# 'http://secif.ipsl.fr/images/SECIF/documents/Communication/fiche_invulnerable/RC_indicateur_EN.pdf'
+###############################################################################
+documentation:
+  description: |
+     Diurnal Temperature Variation (DTR) Indicator and Wind Capacity Factor
+
+  authors:
+    - hunt_al
+    - manu_ni
+    - lled_ll
+    - caro_lo
+    - bojo_dr
+    - gonz_nu
+
+  projects:
+    - c3s-magic
+
+datasets:
+  - {dataset: IPSL-CM5A-LR,  type: exp, project: CMIP5,  mip: day,  exp: historical,  ensemble: r1i1p1,  start_year: 1980,  end_year: 2005}
+
+
+latlon_roi: (/latmin,latmax,lonmin,lonmax/)
+preprocessors:
+  preproc:
+    regrid:
+      target_grid: IPSL-CM5A-LR
+      scheme: linear
+    mask_fillvalues:
+      threshold_fraction: 0.95
+    extract_region:
+      start_longitude: -10
+      end_longitude: 40
+      start_latitude: 27
+      end_latitude: 70
+    extract_season:
+      season: djf
+
+diagnostics:
+  capacity_factor:
+    description: Calculate the wind power capacity factor.
+    variables:
+      sfcWind:
+        preprocessor: preproc
+        mip: day
+        field: T2M
+    scripts:
+      main:
+        script: magic_bsc/capacity_factor.r #(in diag_scripts)
+        power_curves:
+          - /home/Earth/llledo/Documents/Power_Curves/Windographer_library/Enercon_E70_2.3MW.wtp
```

Comment:
Where can these files be downloaded?

Reply:
Dear @bouweandela ,
I'm working on the capacity factor files. About the recipe_combined_indices_wp6.yml has been updated to avoid StartR. It should work know.
Thanks.

Reply:
Dear @bouweandela ,
I've updated the capacity factor recipes. I think that now the files are accessibles for ESMValTool. Furthermore, provenance has been added to the diagnostic. Thanks.